src/tabs/tab.jsx

```diff
@@ -110,20 +111,12 @@ const Tab = React.createClass({
     const textColor = selected ? this.state.muiTheme.tabs.selectedTextColor : this.state.muiTheme.tabs.textColor;
 
     const styles = this.mergeStyles({
-      display: 'table-cell',
-      cursor: 'pointer',
-      textAlign: 'center',
-      verticalAlign: 'middle',
       paddingTop: (icon && !label) ? 4 : 0,
       height: (label && icon) ? 72 : 48,
       color: textColor,
-      outline: 'none',
-      fontSize: 14,
```

Comment:
Can we add it back?
![capture d ecran 2016-01-28 a 11 37 12](https://cloud.githubusercontent.com/assets/3165635/12642022/82f90c80-c5b3-11e5-8fbe-56cc82e0306f.png)